src/Services/TodosContext.tsx

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable max-len */
+/* eslint-disable @typescript-eslint/no-unused-vars */
```

Comment:
Don't disable eslint. Try to fix errors instead

Reply:
I cannot fix eslint-disable @typescript-eslint/no-unused-vars, because such implementation was shown in the video by mentor, and there were no errors, this is esLint problem, not mine

Upd: max-len fixed